modules/ROOT/pages/intro-configuration.adoc

```diff
@@ -3,23 +3,33 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```

Comment:
Curious about the Level-1 title, = Introduction to Mule 4: Configuring Mule Apps.
This is sort of two different things: is the following content a whole intro, or does it focus only on configuring apps?

Reply:
In fact, I wonder if this topic is really focused on Advantages of Configuring Apps Using Mule 4?
Or something about using .yaml files rather than .properties files?

Is the point that although you can continue to use the .properties configuration method used in Mule 3, it is better to use the .yaml config offered in Mule 4, for the following reasons (and list them)?

Take a look at the ideas for alternative wordings below (and especially be sure that they do not alter technical meaning), but also look again at the flow of the narrative. We might be able to minimize this considerably by focusing explicitly on the advantages of .yaml and not get in to the Spring thing or other past-oriented considerations.

Reply:
...and now I reread your edit request and see that the focus is really...:

in Mule 3.x when you create a new application, Studio automatically adds a mule-app.properties file while in Mule 4.x this file is not created anymore. 
Clients can now create a yaml or properties file and both have to be configured using the Global Elements Configuration Properties.

So...should we shorten all of this to:

Mule 4 no longer automatically creates the application properties file that is created in Mule 3.
Instead, you have to manually config app properties or a YAML file 
The YAML file has advantages
But should you continue with properties or use yaml, you have to configure using Global Elements.
And here is the syntax for each option